src/ballet/sha256/fd_sha256.h

```diff
@@ -61,6 +62,9 @@ fd_sha256_align( void );
 FD_FN_CONST ulong
 fd_sha256_footprint( void );
 
+// Synchronization functions
+// =========================
```

Comment:
The codebase doesn't use // comments much. I think dividers like https://github.com/firedancer-io/firedancer/blob/b2088f1f3ab7362e48233636e16b162a74c99d91/src/util/tmpl/fd_map.c#L276 are more in the right style